### PR TITLE
Feature/flaky tests fix

### PR DESCRIPTION
**Pull request by dessap**
### Change description ###
Fix for Scenario Outline: Warning message to display when Case Admin tries to navigate away from booking. 
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite.AcceptanceTests/Helpers/PersonHelper.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
+namespace AdminWebsite.AcceptanceTests.Helpers
+{
+    public class PersonHelper
+    {
+        public PersonHelper()
+        {
+        }
+    }
+}
```
Comment (rehmanab) [inline AdminWebsite/AdminWebsite.AcceptanceTests/Helpers/PersonHelper.cs]: Empty class , please remove

File: AdminWebsite/AdminWebsite.AcceptanceTests/Helpers/HearingsHelper.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
+namespace AdminWebsite.AcceptanceTests.Helpers
+{
+    public class HearingsHelper
+    {
+        public HearingsHelper()
+        {
+        }
+    }
+}
```
Comment (rehmanab) [inline AdminWebsite/AdminWebsite.AcceptanceTests/Helpers/HearingsHelper.cs]: Empty class, please remove

Comment (dessap) [reply AdminWebsite/AdminWebsite.AcceptanceTests/Helpers/HearingsHelper.cs]: done

File: AdminWebsite/AdminWebsite.AcceptanceTests/Helpers/PersonHelper.cs (hunk 1 above)
Comment (dessap) [reply AdminWebsite/AdminWebsite.AcceptanceTests/Helpers/PersonHelper.cs]: done

**Pull request merged**